examples/models/llama2/runner/targets.bzl

```diff
@@ -29,6 +29,7 @@ def define_common_targets():
             ],
             # qnn_executorch_backend can be added below //executorch/backends/qualcomm:qnn_executorch_backend
             exported_deps = [
+                "//executorch/backends/qualcomm:qnn_executorch_backend",
```

Comment:
Just FYI.
I encountered build-error due to this line, but I think it's possibly environment setup issue at my side... don't know how to install "ANDROID" for buck2.

```
  Caused by:                                                                                                                                                          [0/591]
      0: Error looking up configured node root//backends/qualcomm:qnn_executorch_backend (prelude//platforms:default#904931f735703749)
      1: looking up unconfigured target node `root//backends/qualcomm:qnn_executorch_backend`
      2: Error loading targets in package `root//backends/qualcomm` for target `root//backends/qualcomm:qnn_executorch_backend`
      3: From load at backends/qualcomm/TARGETS:2
      4: Error evaluating module: `root//backends/qualcomm/targets.bzl`
      5: error: Module has no symbol `ANDROID`
          --> backends/qualcomm/targets.bzl:3:5
           |
         3 |     "ANDROID",
           |     ^^^^^^^^^
           |



  CMake Error at build/Utils.cmake:216 (message):
    executorch: source list generation failed
  Call Stack (most recent call first):
    CMakeLists.txt:340 (extract_sources)

```

Reply:
I also encountered this error in this PR.
I bypassed it by commenting out this line.

Reply:
I remove it in the latest commit. Sorry for the inconvenience.

Reply:
No worry at all. I'm thinking if we should set up buck2 environment internally.
Is the buck2 flow intended for open-source project?

Reply:
Ah no…we want to remove buck dependency in oss flow…